ogc/features/templates/feature.go.html

```diff
@@ -1,10 +1,47 @@
 {{- /*gotype: github.com/PDOK/gokoala/engine.TemplateData*/ -}}
 {{define "content"}}
+{{ $cfg := .Config }}
+{{ $baseUrl := $cfg.BaseURL }}
+{{ $collId := .Params.CollectionID }}
+
+<script>
+
+    function crsChange(value) {
+        const url = new URL(window.location.href);
+        url.searchParams.set('crs', value);
+        window.location.href = url.toString();
+    }
+
+</script>
+
 <hgroup>
-    <h2 class="title">{{ .Config.Title }} - {{ if and .Params.Metadata .Params.Metadata.Title }}{{ .Params.Metadata.Title }}{{ else }}{{ .Params.CollectionID }}{{ end }}</h2>
+    <h2 class="title">{{ .Config.Title }} - {{ if and .Params.Metadata .Params.Metadata.Title }}{{
+        .Params.Metadata.Title }}{{ else }}{{ .Params.CollectionID }}{{ end }}</h2>
 </hgroup>
 
-<section class="row row-cols-4 g-4 py-3">
+<section class="row row-cols-4 g-4 py-3" id="body" hidden="hidden">
+
+    <div class="col-8">
+        <table class="table table-borderless table-sm w-auto">
+            <tbody>
+            <tr>
+                <td class="w-auto text-nowrap">
+                    <b>{{ i18n "Projection" }}</b>
```

Comment:
Geldt ook voor de features pagina (waar Dominiek) aan werkte aangezien het dezelfde code is (dus als we dit aanpassen op beide plekken doen, of in vervolg ticket). Maar dit is niet WACG compatible. Er wordt nu een tabel gebruikt voor layout, ee de label van het formulier element (de selectiebox) is daardoor semantisch niet correct. Het moet zoiets worden: https://getbootstrap.com/docs/5.3/forms/form-control/#example met gebruik van de `<label>` tag.